src/scrapers/base-isracard-amex.ts

```diff
@@ -306,10 +363,17 @@ async function fetchAllTransactions(page: Page, options: ExtendedScraperOptions,
   });
 
   const accounts = Object.keys(combinedTxns).map((accountNumber) => {
-    return {
+    const account: TransactionsAccount = {
       accountNumber,
       txns: combinedTxns[accountNumber],
     };
+    if (creditUtilization && creditUtilization[accountNumber]) {
+      account.credit = {
+        creditUtilization: creditUtilization[accountNumber].creditUtilization,
+        creditLimit: creditUtilization[accountNumber].creditLimit,
+      };
+    }
```

Comment:
What do you think about this?
```suggestion
    account.credit = creditUtilization?.[accountNumber];
```

1. The `creditUtilization[accountNumber]` structure is the same as the `account.credit`. Even if it is `null` or `undefined` it is OK because we accept no response for some accounts.

Reply:
Looks like a better idea, Will update.